x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/ConnectorTestUtils.java

```diff
@@ -196,9 +196,22 @@ private static FilteringValidation getRandomFilteringValidationError() {
             .build();
     }
 
+    public static FilteringRule getDefaultFilteringRule(Instant currentTimestamp, Integer order) {
```

Comment:
This is accessed from within non test code, so maybe let's move it to `ConnetorFiltering.java` file?

Also we call it a lot with `getDefaultFilteringRule(null, 0)` so we can use method overloading or add `WithOrder` explicitly at the end of the method name:
```
public static FilteringRule getDefaultFilteringRuleWithOrder(Integer order) {
   getDefaultFilteringRule(null, order)
}
```